src/DocumentView/DocumentView.js

```diff
@@ -90,6 +90,11 @@ export default class DocumentView extends PureComponent {
     hideViewModeItems: PropTypes.array,
     pageStackEnabled: PropTypes.bool,
     showQuickNavigationButton: PropTypes.bool,
+    annotationsListEditingEnabled: PropTypes.bool,
+    conversionOptions: PropTypes.string,
+    navigationListForLargeDevices: PropTypes.bool,
+    restrictDownloadUsage: PropTypes.bool,
```

Comment:
@JamesTron same with this one, do you have suggestion for better naming?

Reply:
@sgong-pdftron I think we discussed all of these names, right? I think these are all OK.